Update ISSUE_TEMPLATE (#8151)
vaadin/flow: .github/ISSUE_TEMPLATE.md

<!--
Please read these instructions through before submitting an issue, Thank You!

Only use GitHub issues for bugs and feature requests.   
For general support from the community, use https://vaadin.com/forum or https://gitter.im/vaadin/flow instead.

NOTE: Issues concerning certain component should go to the corresponding components repository like [vaadin/vaadin-grid-flow](https://github.com/vaadin/vaadin-grid-flow).

For feature requests, always include your use case - what are you trying to achieve and why.

Please use the following template to report BUGS:
In case you don't use the template, we might not start working on the issue.
-->
### Description of the bug
<!-- Explain briefly what is broken or what you want to achieve -->
### Minimal reproducible example
<!-- What are the steps to reproduce the issue -->
### Expected behavior
<!-- What should happen -->
### Actual behavior
<!-- What actually happens, attach server/browser logs when there are errors/exceptions -->
### Versions:
    - Vaadin / Flow version:
    - Java version:
    - OS version:

>     - Browser version (if applicable):

Author: ```suggestion
    - Browser version (if applicable):
    - Application Server (if applicable):
    - IDE (if applicable):
```

<!--
Please read these instructions through before submitting an issue, Thank You!

Only use GitHub issues for bugs and feature requests.   
For general support from the community, use https://vaadin.com/forum or https://gitter.im/vaadin/flow instead.

NOTE: Issues concerning certain component should go to the corresponding components repository like [vaadin/vaadin-grid-flow](https://github.com/vaadin/vaadin-grid-flow).

For feature requests, always include your use case - what are you trying to achieve and why.

> Please use the following template to report BUGS:
> In case you don't use the template, we might not start working on the issue.

Reviewer: ```suggestion
Good quality bug report increases the likelihood to get the bug fixed. Please use the following template to report bugs.```

-->
### Description of the bug
<!-- Explain briefly what is broken or what you want to achieve -->
### Minimal reproducible example

> <!-- What are the steps to reproduce the issue -->

Reviewer: ```suggestion
<!-- What are the steps to reproduce the issue, example project or a code snippet without dependencies -->
```
